src/Confluent.Kafka/Timestamp.cs

```diff
@@ -27,6 +27,11 @@ namespace Confluent.Kafka
     public struct Timestamp
     {
         /// <summary>
+        ///     Unix epoch as UTC DateTime
```

Comment:
Maybe: "Unix time is defined as the number of seconds past this UTC time, excluding leap seconds."